static/app/styles/prism.tsx

```diff
@@ -38,7 +38,7 @@ export const prismStyles = (theme: Theme) => css`
     font-family: ${theme.text.familyMono};
     direction: ltr;
     text-align: left;
-    white-space: pre;
+    white-space: pre-wrap;
```

Comment:
@vuluongj20 I think `pre-wrap` is pretty harmless (and usually desirable) but what do you think? Should I isolate this in some other way?

Reply:
I think as a general rule we shouldn't allow code snippets to wrap. It's important to know exactly when a new line starts, and a wrapped line might be misinterpreted as a new line. That's why most docs and in-app code snippets (e.g. GitHub code views) just overflow without wrapping.
<img width="760" alt="image" src="https://github.com/getsentry/sentry/assets/44172267/099c7a3e-fb2f-495c-8c35-7cc2c6c28e96">

We can make Starfish database queries exceptions to this rule. In your screenshots the snippets only span one line, so there's less risk of user confusion (between an actual new line and a wrapped line) here.

Reply:
Understood! I'll roll that back